src/main/resources/upgrade-files/post-upgrade/sql/UAF-3618_add_parm_values_to_DB_upgrade.sql

```diff
@@ -35,4 +35,19 @@ WHERE NMSPC_CD = 'KUALI-TAX';
 
 UPDATE KRIM_PERM_ATTR_DATA_T
 SET ATTR_VAL='KFS-TAX'
-WHERE ATTR_VAL='KUALI-TAX';
\ No newline at end of file
+WHERE ATTR_VAL='KUALI-TAX';
+
+UPDATE KRIM_PERM_ATTR_DATA_T
+SET ATTR_VAL='edu.arizona.kfs.module.tax.document.web.struts.ElectronicFilingReportAction'
+WHERE PERM_ID='11293';
+
+UPDATE KRIM_PERM_T
+SET NM = REPLACE(NM, 'KUALI-TAX', 'KFS-TAX')
+WHERE NM LIKE '%KUALI-TAX%';
+
+UPDATE KRIM_PERM_T
+SET DESC_TXT = REPLACE(DESC_TXT, 'KUALI-TAX', 'KFS-TAX')
+WHERE DESC_TXT LIKE '%KUALI-TAX%';
+
+INSERT INTO KRCR_CMPNT_T (NMSPC_CD, CMPNT_CD, VER_NBR, NM, ACTV_IND)
```

Comment:
Should be:
INSERT INTO KRCR_CMPNT_T (NMSPC_CD, CMPNT_CD, OBJ_ID, VER_NBR, NM, ACTV_IND)
VALUES ('KFS-TAX', 'PayeeMasterExtractStep', SYS_GUID(), 1, 'Payee Master Extract Step', 'Y'); 
OBJ_ID is a non-nullable indexed field, so we need to give it a value.